src/upgrade.js

```diff
@@ -116,13 +118,12 @@ const applyPatch = async (
 /**
  * Upgrade application to a new version of Apollos.
  */
-async function upgrade({ from: fromVersion, to: toVersion, platform, projectName, packageName }) {
+async function upgrade({ from: fromVersion, platform, projectName, packageName }) {
   const tmpPatchFile = `tmp-upgrade-apollos-${platform}.patch`;
-  const projectDir = '.';
+  const projectDir = '.'; 
```

Comment:
I question this line in general. This `projectDir` var is never used.

Reply:
👍